posthog/apps.py

```diff
@@ -15,6 +16,20 @@ class PostHogConfig(AppConfig):
     def ready(self):
         posthoganalytics.api_key = "sTMFPsFhdP1Ssg"
         posthoganalytics.personal_api_key = os.environ.get("POSTHOG_PERSONAL_API_KEY")
+
+        # Skip plugin sync in manage.py scripts and in tests
+        # (the database tables might not yet be created)
+        if (
```

Comment:
The issue was that running `migrate` on a fresh install would fail, as the sync method expects something to be in the database. What could I add to clarify this?

Reply:
I meant trying to run this every boot - worried that it blow up/cause issues in exciting in other places. But this is just FUD and I don't have a solid suggestion - let's resolve for now.